app/mixins/wds-on-site-notifications-unread-count.js

```diff
@@ -0,0 +1,14 @@
+import { gt } from '@ember/object/computed';
+import Mixin from '@ember/object/mixin';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default Mixin.create({
+	wdsOnSiteNotifications: service(),
+
+	unreadCount: computed('wdsOnSiteNotifications.model.unreadCount', function () {
```

Comment:
hmm maybe these two could live inside the service as well?

Reply:
Good idea :+1: Done in https://github.com/Wikia/design-system/pull/236/commits/018ed49e979791ae8fcc417776bcc56e60dc050a